### PR TITLE
Create fallback wrappers for backwards compatibility

### PR DESCRIPTION
**Pull request by andizer**

### CODE REVIEW

File: classes/class-admin-page.php [hunk 1]
```diff
@@ -71,8 +69,121 @@ public function display() {
...
+class WPSEO_News_Wrappers {
+
+	/**
+	 * Fallback for admin_header
+	 *
+	 * @param bool   $form
+	 * @param string $option_long_name
+	 * @param string $option
+	 * @param bool   $contains_files
+	 */
```
Comment (Petervw) [inline classes/class-admin-page.php]: Missing @return phpdoc.

File: classes/class-admin-page.php [hunk 2]
```diff
@@ -71,8 +69,121 @@ public function display() {
...
+
+		return self::admin_pages()->admin_header( true, 'yoast_wpseo_news_options', 'wpseo_news' );
+	}
+
+	/**
+	 * Fallback for admin_footer
+	 *
+	 * @param bool $submit
+	 * @param bool $show_sidebar
+	 */
```
Comment (Petervw) [inline classes/class-admin-page.php]: Missing @return phpdoc.

File: classes/class-admin-page.php [hunk 3]
```diff
@@ -71,8 +69,121 @@ public function display() {
...
+		return self::admin_pages()->admin_footer( $submit, $show_sidebar );
+	}
+
+	/**
+	 * Fallback for the textinput method
+	 *
+	 * @param string $var
+	 * @param string $label
+	 * @param string $option
+	 */
```
Comment (Petervw) [inline classes/class-admin-page.php]: Missing @return phpdoc.

File: classes/class-admin-page.php [hunk 4]
```diff
@@ -71,8 +69,121 @@ public function display() {
...
+
+	/**
+	 * Wrapper for select method.
+	 *
+	 * @param string $var
+	 * @param string $label
+	 * @param array  $values
+	 * @param string $option
+	 */
+	public static function select( $var, $label, $values, $option = '' ) {
```
Comment (Petervw) [inline classes/class-admin-page.php]: Missing @return phpdoc.

File: classes/class-admin-page.php [hunk 5]
```diff
@@ -71,8 +69,121 @@ public function display() {
...
+	}
+
+	/**
+	 * Wrapper for checkbox method
+	 *
+	 * @param        $var
+	 * @param        $label
+	 * @param bool   $label_left
+	 * @param string $option
+	 */
```
Comment (Petervw) [inline classes/class-admin-page.php]: Missing @return phpdoc.

**Comment by Petervw** [comment]
CR1 Done.

**Comment by omarreiss** [comment]
Acceptance testing :heavy_check_mark: I will merge as soon as build passes.

**Pull request merged**